deviceModeInit.js

```diff
@@ -159,20 +160,18 @@ var rudderTracking = (function () {
 
   function checkCartNeedsToBeUpdated(cart) {
     const { attributes } = cart;
-    if (attributes?.rudderAnonymousId) {
-      return false;
-    }
-    return true;
+    return !(attributes?.rudderAnonymousId);
   }
-
-  function updateTimeStampForIdentifierEvent() {
-    const cookieOptions = {
-      action: "set",
-      expire_hr: 2,
-      name: "rs_shopify_cart_identified_at",
-      value: `${Date.now()}`
-    }
-    cookie_action(cookieOptions);
+  function sendRudderIdentifierPeriodically() {
+    getAndSendIdentifierToRudderWebhook();
+    const cookieUpdateFixedInterval = 50 * 60 * 1000; // 50 mins
```

Comment:
I thought we decided on much lower time of 5 mins or something

Reply:
We don't need pool time any.more that's the  time after which the rudderIdentifier event will fire automatically periodically